Standards/scs-xxxx-v1-sovereign-cloud-standards-yaml.md

```diff
@@ -0,0 +1,151 @@
+| `name` | String | Full name of this certification level	| _SCS Open_ |
+| `url` | String | Valid URL to the latest raw version of this document | _https://raw.githubusercontent.com/SovereignCloudStack/Docs/main/Certification/scs-open.yaml_ |
+
+The certification YAML *MAY* contain the following keys:
+
+## Dependency
+
+Standards that are required by lower certification levels shouldn't be included in higher tier certfications again. We thus need to note
+on which certification this level is depending on.
+
+| Key 	| Type 	| Description 	| Example   |
+|-----	|------	|-------------	|---------- |
+| `depends_on` | Map | Preliminary certification level on which this certification depends | |
+| `depends_on.name` | String | Full name of the depending certification level | _SCS Compatible_ |
+| `depends_on.url` | String | Valid URL to the latest raw version of the depending certification level | _https://raw.githubusercontent.com/SovereignCloudStack/Docs/main/Certification/scs-compatible.yaml_ |
+
+## Layers
+
+The Sovereign Cloud Stack is compiled of several layers: Infrastructure, Container, IAM and Operations. Every layer may include different
+mandatory or optional standards. Every layer is noted as a dedicated map in the certification YAML.
+
+| Key 	| Type 	| Description 	| Example   |
+|-----	|------	|-------------	|---------- |
+| `iaas` | Map | List of versioned sets of SCS standards for the IaaS layer  | |
+| `kaas` | Map | List of versioned sets of SCS standards for the KaaS layer  | |
+| `iam` | Map | List of versioned sets of SCS standards for the IAM layer  | |
+| `ops` | Map | List of versioned sets of SCS standards for the Ops layer  | |
+
+Every layer map keeps track of the particular SCS standards for this layer and their history over time.
+
+| Key 	| Type 	| Description 	| Example   |
+|-----	|------	|-------------	|---------- |
+| `{layer}.stable` | String | Version of stable set of standards for this layer | _v3_ |
+| `{layer}.stabilized_at` | Date | ISO formatted date indicating the date after which the set of standards was considered stable | _2022-11-09_ |
+| `{layer}.upcoming` | String | Version of upcoming set of standards for this layer | _v4_ |
+| `{layer}.replaced_at` | Date | ISO formatted date indicating the date on which the upcoming set of standards will replace the current stable | _2023-04-09_ |
+| `{layer}.versions` | Array of maps | List of all past and upcoming set of standards | |
+
+## Set of standards
+
+| Key 	| Type 	| Description 	| Example   |
+|-----	|------	|-------------	|---------- |
+| `{layer}.versions.v{N}` | Map | Versioned set of standards for this particular layer  | |
+
+## Basic Example
+
+```yaml
+name: SCS Open
+url: https://raw.githubusercontent.com/SovereignCloudStack/Docs/main/Certification/scs-open.yaml
+depends_on:
+  name: SCS Compatible
+  url: https://raw.githubusercontent.com/SovereignCloudStack/Docs/main/Certification/scs-compatible.yaml
```

Comment:
if no minimal version is specified, any version is accepted?

Reply:
The minimal versions should be noted within the corresponding layers. This is just for building a dependency graph.